lib/index.js

```diff
@@ -26,7 +26,7 @@ class EvervaultClient {
       crypto.createECDH(this.config.encryption.ecdhCurve)
     );
 
-    if (options.intercept !== false) {
+    if (options.intercept !== false && options.relay !== false) {
```

Comment:
Think we should include a deprecation warning if they're still using `options.relay`? This still isn't fully backwards compatible btw, cause previously if someone didn't have `options.relay` set the default was no interception. But if they continue to have nothing the default will be to intercept. Don't think it can be made fully backwards compatible while changing the default